src/components/Popover/Popover.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+
+import { Button, GitHub, Menu } from '../../index';
+import Popover from './Popover';
+
+describe('Popover', () => {
+  test('default snapshot', async () => {
+    const content = (
+      <Menu>
+        <Menu.Item name="default">Option 1</Menu.Item>
+        <Menu.Item prefixIcon={<GitHub />}>Option 2</Menu.Item>
+      </Menu>
+    );
+
+    const component = (
+      <Popover isVisible onClose={() => {}} content={content}>
+        <Button>Click me!</Button>
+      </Popover>
+    );
+    const { asFragment, getByTestId } = render(component);
+
+    await waitFor(() => getByTestId('menu-item-default'));
```

Comment:
Is it possible to write it without `await waitFor()`?

Reply:
No, it complains about an update that has to be wrapped in "act"